package.json

```diff
@@ -34,7 +34,8 @@
     "test-browser": "node tests/browser",
     "test-cli": "nodeunit tests/cli.js",
     "test-node": "npm run test-unit && npm run test-cli && npm run test-regression",
-    "test-regression": "nodeunit tests/regression",
+    "test-regression": "nodeunit tests/regression && node bin/jshint src",
+    "test-rhino": "java -jar tests/regression/libs/rhino-1.7.7.jar -opt -1 dist/jshint-rhino.js src",
```

Comment:
Yes, it "only" runs jshint over itself.  Without reimplementing `nodeunit` for rhino, I don't believe that the unit tests can be run.  It would certainly be nicer to run the unit tests (a future patch, perhaps!) but this way we at least have *some* idea of whether the rhino build is still roughly working.  It will catch most problems of the #2853 type, where the rhino build fails to start up at all.